DTS-STN/Alpha-Site
# Feature/about page

# Description

[Add "About" Page](https://trello.com/c/YGrk0Yim/167-add-about-page)

![image](https://user-images.githubusercontent.com/31868510/117690091-7619c100-b188-11eb-99f7-ab7446831fcf.png)

Created and added About page to navigation. 

## Acceptance Criteria

The site has an "About" page that can be navigated to through the site navigation and is fully translated and tested.

## Test Instructions

1. go to homepage
2. select "About" from the navigation menu
3. see that page loads without errors

## Help Requested

- I could not figure out how to setup the image to match the figma designs precisely, any help there would be greatly appreciated. I assume it will behave similarly to the image at the top of the "Become a tester" form in figma if the content of the page expands. (cough do we need these images cough)

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
